api/ocs/test/test_ocs_api.py

```diff
@@ -498,7 +503,7 @@ def create_minimodel_program() -> Program:
             qa_state=QAState.NONE,
             guide_state=False,
             resources={gmosn},
-            wavelength={0.475}
+            wavelengths={0.475}
```

Comment:
I think we might have maybe tread on each others toes on these changes since we did similar things? I guess we'll find out during merge time!